### PR TITLE
simplify logcatd

### PR DESCRIPTION
**Pull request by deanlee**

### CODE REVIEW

**Review by Ashjess — APPROVED** [review]

File: selfdrive/logcatd/logcatd_systemd.cc [hunk 1]
```diff
@@ -36,17 +36,8 @@ int main(int argc, char *argv[]) {
...
-        assert(r >= 0);
-      } while (r == SD_JOURNAL_NOP && !do_exit);
-
-      if (do_exit) break;
-
-      r = sd_journal_next(journal);
-      assert(r >= 0);
-
-      if (r == 0) continue; // Try again if we still didn't get anything
+      sd_journal_wait(journal, 1000 * 1000);
```
Comment (pd0wm) [inline selfdrive/logcatd/logcatd_systemd.cc:39]: Did you mean?
```suggestion
      r = sd_journal_wait(journal, 1000 * 1000);
```

Comment (deanlee) [reply selfdrive/logcatd/logcatd_systemd.cc]: maybe an assert is better.

Comment (pd0wm) [reply selfdrive/logcatd/logcatd_systemd.cc]: yes. You probably saw my initial comment before I edited it.

**Comment by pd0wm** [comment]
opendbc is also changed. can you revert that?

File: selfdrive/logcatd/logcatd_systemd.cc (hunk 1 above)
Comment (deanlee) [reply selfdrive/logcatd/logcatd_systemd.cc]: yes.lol.

**Comment by deanlee** [comment]
Done

**Pull request merged**